views/default/forms/usersettings/save.php

```diff
@@ -5,10 +5,18 @@
  * Plugins should extend "forms/account/settings" to add to the settings.
  */
 
-$form_body = elgg_view("forms/account/settings", $vars);
+$user = elgg_get_page_owner_entity();
 
-$form_body .= '<div class="elgg-foot">';
-$form_body .= elgg_view('input/submit', array('value' => elgg_echo('save')));
-$form_body .= '</div>';
+$form_body = elgg_view('forms/account/settings', $vars);
+
+$submit = elgg_view('input/submit', ['value' => elgg_echo('save')]);
+$hidden = '';
+
+if ($user) {
+	// need the user's guid to make sure the correct user gets updated
```

Comment:
This is required only for admins, right? Something like this would be more descriptive:

> The user defaults to the logged in user, so we need to include the
> user GUID so that admins can edit the settings of other users